configuration/concepts/registration_concepts.csv

```diff
@@ -0,0 +1,42 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+050ecb0e-fcc7-4778-94ee-3258ddecbc60,,,Haiti_Single,Haiti_Single,Single,Célibataire,,,Misc,N/A,,,,,,,,,,,
+7dd0e065-818b-43c7-8344-9064accf97f5,,,Haiti_Married,Haiti_Married,Married,Marrié,,,Misc,N/A,,,,,,,,,,,
+c064ecf5-a6ea-48ce-807d-a8e554c13f8d,,,Haiti_Living_with_Partner,Haiti_Living_with_Partner,Living with Partner,En ménage,,,Misc,N/A,,,,,,,,,,,
+176a6391-e203-4d7e-b0e1-9baf63602a91,,,Haiti_Separated,Haiti_Separated,Separated,Séparé,,,Misc,N/A,,,,,,,,,,,
+7f980dba-41c2-42a5-9ddc-ccffdc881a38,,,Haiti_Divorced,Haiti_Divorced,Divorced,Divorcé,,,Misc,N/A,,,,,,,,,,,
+73515be9-3f0e-4844-b28b-12967968e477,,,Haiti_Widowed,Haiti_Widowed,Widowed,Veuf(ve),,,Misc,N/A,,,,,,,,,,,
+4e7a07f2-345f-419c-8d30-b060d9b8681a,,,Haiti_Social_Status,Haiti_Social_Status,Social Status,Situation maritale,,,Misc,Coded,,Haiti_Single; Haiti_Married; Haiti_Living_with_Partner; Haiti_Separated; Haiti_Divorced; Haiti_Widowed,,,,,,,,,
+7c47a358-1d99-4047-9343-ca5f1948a848,,,Ivory_Coast_Agriculteur,Ivory_Coast_Agriculteur,Agriculteur,Agriculteur,,,Misc,N/A,,,,,,,,,,,

```

Comment:
Prefix would rather be `EasyDoc`

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/5/commits/307d23dae554f429560422981ff6daf068473556)